plugins/jvm-example/src/test/java/org/elasticsearch/plugin/example/ExampleExternalIT.java

```diff
@@ -29,9 +27,18 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 
-public class ExampleExternalIT extends ESTestCase {
-    public void testExample() throws Exception {
-        String stringAddress = Objects.requireNonNull(System.getProperty("external.address"));
+import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.test.ESClientTestCase;
+
+import static org.hamcrest.Matchers.greaterThan;
+
+/**
+ * Example integration test against an external client.
+ */
+public class ExampleExternalIT extends ESClientTestCase {
```

Comment:
I don't understand this change...the test is for an external fixture, not a client.